packages/transport/src/modules/sync/externalEventProcessors/RelationshipExternalEventProcessor.ts

```diff
@@ -0,0 +1,17 @@
+import { RelationshipChangedEvent } from "../../../events";
+import { Relationship } from "../../relationships/local/Relationship";
+import { ExternalEventProcessor } from "./ExternalEventProcessor";
+
+export abstract class RelationshipExternalEventProcessor extends ExternalEventProcessor {
+    protected triggerRelationshipChangedEvent(changedRelationship: Relationship, oldRelationship?: Relationship): void {
+        if (!this.didChangeRelationship(changedRelationship, oldRelationship)) return;
+
+        this.eventBus.publish(new RelationshipChangedEvent(this.ownAddress, changedRelationship));
+    }
+
+    private didChangeRelationship(changedRelationship: Relationship, oldRelationship?: Relationship): boolean {
```

Comment:
The function name makes me think of dids. 😆 Maybe rename it to `changedRelationship` or `relationshipWasChanges` (however this doesn't sound as good with `this.` before) or something else. Or just leave it if you don't consider it to be important.

Reply:
changed it

Reply:
Sure, I just stumbled across this. Also, because one wouldn't use "did" here and just use past tense, unless to put an emphasize on this. But if that is what you're aiming at, it's all good.